packages/styles/scss/ibm-dotcom-styles.scss

```diff
@@ -43,6 +43,7 @@
 @import 'patterns/blocks/content-group-simple/index';
 @import 'patterns/blocks/feature-card-block-medium/feature-card-block-medium';
 @import 'patterns/blocks/logo-grid/logo-grid';
+@import 'patterns/blocks/leadspace-block/index';
```

Comment:
See comment at https://github.com/carbon-design-system/ibm-dotcom-library/pull/2391/files#r426837520.

The `index` file may not seem necessary as you are already pulling in the necessary imports in `leadspace-block`, including the global import.

Reply:
yeah removed the index file and importing the leadspace-block here.